Userland/Libraries/LibWeb/CSS/Keywords.json

```diff
@@ -355,6 +355,7 @@
   "sw-resize",
   "swap",
   "table",
+  "tab-size",
```

Comment:
Property names are not keywords, so it doesn't need to be here. A keyword is a value to a property, called `valid-identifiers` in `Properties.json` because I haven't got around to correcting the terminology there yet. (They're also used by media queries.)

Reply:
ah OK, I see were I got confused here. I'll remove this.